styles.css

```diff
@@ -0,0 +1,402 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Rubik:wght@300&display=swap");
+
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans:wght@700&family=Oswald:wght@500;700&family=Roboto:wght@300&family=Rubik:wght@300&display=swap");
+
+html {
+  font-size: 62.5%;
+}
+
+/* Reset CSS */
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  --dark-cyan: darkcyan;
+  --grey: grey;
+  --teal: teal;
+}
+
+.rubik-font {
+  font-family: "Rubik", sans-serif;
+}
+
+.head-font {
+  font-family: "Noto Sans", sans-serif;
+}
+
+/* Navbar */
+.nav-bar {
+  padding: 1rem;
+  height: 12vh;
+  padding: 0.5rem;
+  box-shadow: 1px 2px 10px var(--grey);
+  align-items: center;
+  overflow: hidden;
+}
+
+.header-name {
+  font-size: 2.5rem;
+}
+
+.header-items-left {
+  width: 17rem;
+  margin: 0.5rem 0;
+}
+
+.header-items-right {
+  display: none;
+}
+
+.home-side-bar {
+  align-self: center;
+  font-size: 2rem;
+}
+
+.header-links {
+  text-decoration: none;
+  font-size: 1.4rem;
+  color: var(--dark-cyan);
+  align-self: center;
+  display: none;
+}
+
+.header-links:hover {
+  color: var(--grey);
+}
+
+.header-icon-links {
+  font-size: 1.7rem;
+}
+
+.header-ic-link-description {
+  font-size: 1rem;
+}
+
+.header-input {
+  border-radius: 10px;
+  width: 10rem;
+  height: 3rem;
+  border: 1px solid var(--grey);
+  text-align: right;
+  padding-right: 4px;
+}
+
+.home-side-bar.side-bar-ico {
+  background-color: var(--white);
+}
+
+.header-input:focus {
+  outline-color: var(--dark-cyan);
+}
+
+.header-search-box {
+  position: relative;
+  width: max-content;
+  height: max-content;
+}
+
+.header-search {
+  position: absolute;
+  left: 0.7rem;
+  top: 0.5rem;
+  font-size: 2rem;
+  color: var(--grey);
+}
+
+.header-icons {
+  cursor: pointer;
+}
+
+.header-icons:hover {
+  color: var(--teal);
+}
+
+.top-image-container {
+  overflow: hidden;
+  display: grid;
+}
+
+.top-img {
+  width: 100%;
+  height: 100%;
+  grid-row-start: 1;
+  grid-row-end: 4;
+  grid-column-start: 1;
+  grid-column-end: 4;
+}
+
```

Comment:
its correct, if possible use grid area - grid-area: row-start/col-start/row-end/col-end (if it works! )

Reply:
Ok, will try that.